src/docs/sdk/performance/modules/assets.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: 'Assets Module'
+---
+
+The javascript SDK should automatically instrument most assets fetched by the application. This includes images, stylesheets, scripts, and other assets. But if you may wish to instrument assets manually.
+
+## Span conventions
+
+### Span Operations
+
+| Span OP           | Description                             |
+| :---------------- | :-------------------------------------- |
+| `resource.script` | Javscript was fetched by the browser    |
+| `resource.css`    | A Stylesheet was fetched by the browser |
+| `resource.img`    | An Image was fetched by the browser     |
```

Comment:
You probably don't need the whole table, since the descriptions are all kind of the same. Instead, maybe you can explain the applicable resource types in the intro, and also immediately explain how to name the `op` field of the spans? Or something along those lines

Reply:
updated!